pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/grafana/dashboard-configs/index.md

```diff
@@ -0,0 +1,154 @@
+
+You can put the Grafana dashboard configurations in a folder in the repository.
+For instance, `https://github.com/company/dashboard-configs/production`.
+
+```json
+{
+  "grafana": {
+    "dashboard_config_repository": {
+      "url": "https://github.com/company/dashboard-configs",
+      "path": "/production"
+    }
+  }
+}
+```
+
+The Git repository can be either public or private.
+If the Git repository is private, you will need to configure the credentials to access the Git repository (see below).
+
+<p class="message--note"><strong>NOTE: </strong>If enabled, the set of default dashboards shipped with the service is from the Mesosphere-maintained Git repository
+<code>https://github.com/dcos/grafana-dashboards</code>.
+These dashboards are updated with {{ model.techName }} service releases.
+If you would like to use the most up-to-date version of these dashboards that hasn't yet been released,
+you can configure the repository address to point to <code>https://github.com/dcos/grafana-dashboards</code>.
+The <code>grafana.default_dashboards</code> option should be set to <code>false</code>.
+As it's a public repository, there is no need to set up the `credentials`.</p>
+
+```json
+{
+  "grafana": {
+    "default_dashboards": false,
+    "dashboard_config_repository": {
+      "url": "https://github.com/dcos/grafana-dashboards",
+      "path": "/dashboards"
+    }
+  }
+}
+```
+
+## Create secrets for Git repository credentials
+
+If the Git repository containing the Grafana dashboard configurations is private, you will need to configure the secrets first.
+Currently, the following Auth types are supported.
+
+<p class="message--important"><strong>IMPORTANT: </strong>If the Git repository contains submodules, all the submodules must use the same Auth type as what's used for the repository.
+For example, if HTTP Auth is configured for the Git repository, all of the submodules in the repository's <code>.gitmodules</code> file must use the <code>https</code> URL scheme.</p>
+
+### HTTP Auth
+
+```bash
+dcos security secrets create --value=<GITHUB_USERNAME> githubusername-secret
+dcos security secrets create --value=<GITHUB_PASSWORD> githubpassword-secret
+```
+
+For GitHub, the password can be the API token.
+
+Create a custom option file (`options.json`) like the following.
+You can omit the `credentials` section if the Git repository is public.
+
+```json
+{
+  "grafana": {
+    "dashboard_config_repository": {
+      "url": "https://github.com/company/dashboard-configs",
+      "path": "/production",
+      "credentials": {
+        "username_secret": "githubusername-secret",
+        "password_secret": "githubpassword-secret"
+      }
+    }
+  }
+}
+```
+
+### SSH Auth
+
+```bash
+dcos security secrets create -f <PATH_TO_PRIVATE_KEY> gitsshkey-secret
+```
+
+For GitHub, you must add the [Deployment Key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) (that's the public key) to the repository.
```

Comment:
The change from "that is" -> "that's" here, while probably technically correct, is awkward and we can see Vale also complains.

Please change back to "(that is, the public)", or just "(the public key)